chart/values.yaml

```diff
@@ -586,6 +586,9 @@ workers:
 
   podAnnotations: {}
 
+  # Labels specific to workers
```

Comment:
```suggestion
  # Labels specific to workers objects and pods
```

Should be like this across the board, also in the schema description.

Reply:
done